### PR TITLE
added config param for fixed route aka server

### PR DESCRIPTION
**Pull request by ChaosKid42**
This PR adds a new parameter to fix the route. This essentially means that punjab can be configured to always connect to the same XMPP server.

### CODE REVIEW

File: punjab/session.py [hunk 1]
```diff
@@ -83,8 +83,14 @@ def make_session(pint, attrs, session_type='BOSH'):
     if s.hostname in ['localhost', '127.0.0.1']:
         connect_srv = False
     if not connect_srv:
-        reactor.connectTCP(s.hostname, s.port,
-                           s, bindAddress=pint.bindAddress)
+        if s.port == 5223 and ssl:
```
Comment (twonds) [inline punjab/session.py]: Checking this specific port seems a bit restrictive. What if you want to have ssl but use a different port?

Comment (ChaosKid42) [reply punjab/session.py]: That's indeed more like a quick hack. I removed it from the PR for the time being until I might come up with a better implementation.

**Comment by ChaosKid42** [comment]
The use case I'm thinking of is the following:
I want to separate the bosh connection manager from the actual xmpp-server (e.g. because I want to be able to scale separately. Still punjab is supposed to be the front-end to my xmpp-server only. Additionally I don't want to rely on SRV-record lookups as I don't use an SRV-record in my scenarion.

**Comment by ChaosKid42** [comment]
see https://github.com/twonds/punjab/pull/70

**Comment by twonds** [comment]
I am gonna merge this but I believe the XEP only describes the client providing the route attribute.

**Pull request merged**